frontend/src/components/CategorySelector.js

```diff
@@ -0,0 +1,44 @@
+import {Select, MenuItem, FormControl} from "@material-ui/core"
+import {useState} from "react";
+import useStyles from "./DropdownStyle";
+
+export default function CategorySelector() {
+
+    const style = useStyles();
+    const [category, setCategory] = useState("");
+
+    const handleChange = selection => setCategory(selection.target.value);
+
+    return (
+        <section>
+            <FormControl className={style.formControl}>
+                <Select className={style.Select} disableUnderline={true} onChange={handleChange}>
```

Comment:
i think there should be a possibility to set the selected default value. like value={category}